src/Simplex/Chat/Migrations/M20231219_note_folders.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Migrations.M20231219_note_folders where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20231219_note_folders :: Query
+m20231219_note_folders =
+  [sql|
+    CREATE TABLE note_folders (
+      note_folder_id INTEGER PRIMARY KEY AUTOINCREMENT,
+      user_id INTEGER NOT NULL REFERENCES users ON DELETE CASCADE,
+      display_name TEXT NOT NULL,
```

Comment:
+ image
@epoberezkin do we need images for these?

Reply:
yes